pisa/core/events_pi.py

```diff
@@ -163,7 +167,12 @@ def load_events_file(self, events_file, variable_mapping=None):
                     )
 
         if isinstance(events_file, str):
-            input_data = from_file(events_file)
+            input_data,metadata = from_file(events_file,return_attrs=True)
```

Comment:
is this metedata always present? Asking for backwards compatibility

Reply:
that metadata is always present, at least based on how pisa hdf5 files are created